client/boot/index.js

```diff
@@ -323,7 +323,11 @@ function reduxStoreReady( reduxStore ) {
 			} );
 
 			if ( '/' === context.pathname && config.isEnabled( 'devdocs/redirect-loggedout-homepage' ) ) {
-				page.redirect( '/devdocs/start' );
+				if ( config.isEnabled( 'oauth' ) ) {
+					page.redirect( '/authorize' );
+				} else {
+					page.redirect( '/devdocs/start' );
```

Comment:
Thanks for the feedback. It seems perhaps we can use the new auth page in general, adding some more info for developers (like /start/developer as the create account), and redirecting to devdocs/welcome after the oauth handshake if devdocs is enabled?

I can see us deprecating no-oauth + yes-devdocs as any developer flow should be using oauth in the future. Which means we'd have:

- `/login` for desktop app.
- `/authorize` (or a different name) for oauth, and changing its functionality based on devdocs enabled.
- Get rid of devdocs/start.